### PR TITLE
[ci] Run Meson unit tests in CI

### PR DESCRIPTION
**Pull request by mcy**
https://github.com/lowRISC/opentitan/pull/1433 will wind up changing how this is done anyway but I want to get *some* test execution in now rather than later.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -147,6 +147,12 @@ jobs:
       ninja -C "$(sw_obj_dir sim-verilator)" all
       ninja -C "$(sw_obj_dir fpga)" all
     displayName: 'Build embedded targets'
+  - bash: |
+      # NOTE: Since tests are independent of the underlying target,
+      # we execute only one target for now. This will go away once
+      # we merge #1433.
+      ninja -C "$(sw_obj_dir fpga)" test
```
Comment (gkelly) [inline azure-pipelines.yml]: I think you need a `. util/build_consts.sh` before this in this step.

Comment (mcy) [reply azure-pipelines.yml]: Oops.

**Comment by imphil** [comment]
#1433 is merged now, this needs an update. But beyond that LGTM.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -147,6 +147,13 @@ jobs:
       ninja -C "$(sw_obj_dir sim-verilator)" all
       ninja -C "$(sw_obj_dir fpga)" all
     displayName: 'Build embedded targets'
+  - bash: |
+      # NOTE: Since tests are independent of the underlying target,
+      # we execute only one target for now. This will go away once
+      # we merge #1433.
```
Comment (gkelly) [inline azure-pipelines.yml]: Update comment please.

Comment (gkelly) [reply azure-pipelines.yml]: ^

Comment (mcy) [reply azure-pipelines.yml]: Done.

**Pull request merged**